### PR TITLE
Show Featured Templates on Apps Home

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Show Featured Templates on Apps Home

Remove Training inApp message
Clean up directive and factory

Add Training link to nav menu

[stage-2]

## Motivation and Context
Address review points related to Featured Templates for the epic

## How Has This Been Tested?
Tested changes locally, updated unit tests

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
I've left a lot of the close/persist logic for In App messages in place, in case we want to add other messages that support local storage.

Also, the Training link is static.

@ezequielc please review. Thanks!

File: web/partials/common-header/common-header.html [hunk 1]
```diff
@@ -41,7 +41,10 @@
 							<a id="{{opt.title + 'Link'}}" ng-if="opt.cid" ng-href="{{opt.link}}" link-cid target="{{opt.target}}" ng-class="{'selected': opt.states && opt.states.indexOf(navSelected) > -1}">{{opt.title}}</a>
 							<a id="{{opt.title + 'Link'}}" ng-if="!opt.cid" ng-href="{{opt.link}}" target="{{opt.target}}" ng-class="{'selected': opt.states && opt.states.indexOf(navSelected) > -1}">{{opt.title}}</a>
 						</li>
-						
+            <li>
+              <a id="trainingLink" href="https://risevision.com/webinars?utm_source=app&utm_medium=web&utm_campaign=in_app_training_notification" target="_blank">Training</a>
```
Comment (ezequielc) [inline web/partials/common-header/common-header.html:45]: @alex-deaconu Any reason why you didn't add the link from here: https://github.com/Rise-Vision/rise-vision-apps/blob/master/web/scripts/common/controllers/ctr-app.js#L6 ? Thanks!

Comment (alex-deaconu) [reply web/partials/common-header/common-header.html:45]: @ezequielc yes, a few reasons. First off, the link is static and doesn't require highlighting. Second, if we ever deploy this to RVA the Training link would appear automatically with this change, which I think is a good bonus. I can update if you feel strongly about moving it in the code. Thanks.

Comment (ezequielc) [reply web/partials/common-header/common-header.html:45]: @alex-deaconu I am good with leaving as is. Thanks!

**Pull request merged**